webapp/views.py

```diff
@@ -18,6 +19,77 @@
 )
 
 
+@certification_blueprint.route("/hardware/<hardware_id>")
+def hardware(hardware_id):
+    modelinfo_data = get_device_information_by_hardware_id(hardware_id)
+
+    hardware_details = {}
+    for component in modelinfo_data.get("hardware_details"):
```

Comment:
In the live page, "Other" appears at the bottom of the list, whereas in yours, it's above "Processor", "System" and "USB", presumably because you're sending them through in alphabetical order.

So I guess you need special logic in here to move "Other" to the bottom.